File: 이건희/level1/시저 암호.py
def solution(s, n):
    word = list(s)
    result = []
    for i in word:
        num = ord(i)
        if 65 <= num <= 90:
            num += n
            if 90 < num:
                num -= 26
        elif 97 <= num <= 122:
            num += n
            if 122 < num:
                num -= 26
        result.append(chr(num))
    return ''.join(result)

File: 김찬우/프로그래머스 /Lv.1/시저암호.java
class Solution {
    public String solution(String s, int n) {
        
        char[] charArr = s.toCharArray();  
        
        
        for(int i =0; i<charArr.length;i++){
            
            if(charArr[i]==' '){
                continue;
            }
            
            int result = (int)charArr[i]+n;  
            if(result>90 && charArr[i]<91){
                result = result-90+64;
                
            }else if(result>122 && charArr[i]>96){
                result = result-122+96;
            }
            
            charArr[i] = (char)result;
            
        
            
            
            
            
            
        }
        return String.valueOf(charArr);
    }
}
